intrastat_product/models/intrastat_product_declaration.py

```diff
@@ -239,18 +240,74 @@ def _account_config_warning(self, msg):
             msg, action.id, _("Go to Accounting Configuration Settings screen")
         )
 
-    def _get_partner_country(self, inv_line):
-        country = (
-            inv_line.move_id.src_dest_country_id
-            or inv_line.move_id.partner_id.country_id
-        )
-        if not country.intrastat:
-            country = False
-        elif country == self.company_id.country_id:
-            country = False
```

Comment:
This check is missing in the new changes, so we are generating export/import lines related to the company's own country.